src/rmodels.c

```diff
@@ -5106,6 +5169,173 @@ static Model LoadGLTF(const char *fileName)
 
     return model;
 }
+
+static void GetGLTFPoseAtTime(cgltf_accessor* input, cgltf_accessor *output, float time, void *data) {
+    assert(input->count == output->count);
+
+    float tstart = 0;
+    float tend = 0;
+
+    int keyframe = 0; // defaults to first pose
+    for (int i = 0; i < input->count - 1; i++) {
+        cgltf_bool r1 = cgltf_accessor_read_float(input, i, &tstart, 1);
+        assert(r1);
```

Comment:
Please, could you avoid the `assert()`, no asserts are used in raylib, instead warnings are logged using `TRACELOG()` macro.